runtime/lua/vscode.lua

```diff
@@ -1,11 +1,13 @@
+require("cursor")
```

Comment:
If this is on 'runtimepath' it could conflict with a plugin. To minimize conflicts, choose a single "namespace" and put all modules under it (see previous comment).

```suggestion
require("vscode.cursor")
```